Gemfile.lock

```diff
@@ -1,42 +1,41 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    activesupport (6.0.0)
+    activesupport (6.0.3.2)
       concurrent-ruby (~> 1.0, >= 1.0.2)
       i18n (>= 0.7, < 2)
       minitest (~> 5.1)
       tzinfo (~> 1.1)
-      zeitwerk (~> 2.1, >= 2.1.8)
-    coderay (1.1.2)
-    concurrent-ruby (1.1.5)
-    connection_pool (2.2.2)
-    database_cleaner (1.7.0)
+      zeitwerk (~> 2.2, >= 2.2.2)
+    coderay (1.1.3)
+    concurrent-ruby (1.1.6)
+    connection_pool (2.2.3)
+    database_cleaner (1.8.5)
     diff-lcs (1.3)
     docile (1.3.2)
-    etna (0.1.8)
+    etna (0.1.13)
       jwt
       multipart-post
       net-http-persistent
       rack
-    factory_bot (5.1.1)
+    factory_bot (5.2.0)
       activesupport (>= 4.2.0)
-    i18n (1.7.0)
+    i18n (1.8.3)
       concurrent-ruby (~> 1.0)
-    json (2.2.0)
     jwt (2.2.1)
-    method_source (0.9.2)
+    method_source (1.0.0)
     mini_portile2 (2.4.0)
-    minitest (5.12.2)
+    minitest (5.14.1)
     multipart-post (2.1.1)
-    net-http-persistent (3.1.0)
+    net-http-persistent (4.0.0)
       connection_pool (~> 2.2)
-    nokogiri (1.10.4)
+    nokogiri (1.10.9)
       mini_portile2 (~> 2.4.0)
-    pg (1.1.4)
-    pry (0.12.2)
-      coderay (~> 1.1.0)
-      method_source (~> 0.9.0)
-    rack (2.0.7)
+    pg (1.2.3)
+    pry (0.13.1)
+      coderay (~> 1.1)
+      method_source (~> 1.0)
+    rack (2.2.3)
```

Comment:
So the other apps currently use `rack` `2.2.2` -- is this going to cause the same conflict problem, but "future"-direction wise?